src/test/java/com/opentable/metrics/graphite/GraphiteConnectTest.java

```diff
@@ -111,12 +114,18 @@ public void reconnectTest() throws IOException, InterruptedException {
         final BeanFactory factory = context.getAutowireCapableBeanFactory();
         final Counter detectedConnectionFailures = findConnectionFailureCounter(factory);
         final Counter connectionCloses = findConnectionCloseCounter(factory);
+        final MetricRegistry metricRegistry = factory.getBean(MetricRegistry.class);
+        final Counter counterBoo = metricRegistry.counter("wolf.nipple.chips");
+        counterBoo.inc();
+        counterBoo.inc();
+        counterBoo.inc();
+
 
         // Wait for graphite to connect.
         Thread.sleep(reportingPeriod.multipliedBy(2).toMillis());
         Assert.assertTrue(server.getBytesRead() > 0);
         Assert.assertEquals(0, detectedConnectionFailures.getCount());
-
+        server.contains("wolf created but otherwise default", "foo");
```

Comment:
?

Reply:
I added a sort of TDD like clause soo I could debug.